extensions/amp-story-panning-media/0.1/amp-story-panning-media.md

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
Do you plan to fill this in this PR? We should still submit the correct tags or keep the sections empty if you want to fill later, but no need to explain everything rn

Reply:
This is a placeholder. Agree that we should have correct tags 🙌